src/extremeweatherbench/evaluate.py

```diff
@@ -252,7 +337,9 @@ def _open_obs_datasets(eval_config: config.Config):
     point_obs = None
     gridded_obs = None
     if eval_config.point_obs_path:
-        raise NotImplementedError("Point obs evaluation not implemented as of 0.1.0")
+        point_obs = pd.read_parquet(
+            eval_config.point_obs_path, storage_options=dict(token="anon")
+        )
```

Comment:
`storage_options` should probably be lifted to a module-level parameter that you can define as a sane default, with the explanation that we provide access to a curated set of observations gratis for users.